com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Procedural/CheckerboardNode.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+
+    internal class CheckerboardNode : IStandardNode
+    {
+        public static string Name = "Checkerboard";
+        public static int Version = 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+@"
+{
+#if defined(SHADER_STAGE_RAY_TRACING)
+    int2 checker = frac(Frequency * UV) > 0.5;
```

Comment:
Get a shader error when creating a checkerboard node, is this a known issue? 
![image](https://user-images.githubusercontent.com/13033542/167044799-074eb3ff-11b2-4e86-9f35-0664e03352cf.png)

Reply:
This appears to be a bug in the system that's supposed to add the ports to the nodes.  Frequency is set to be Usage.In - which means it should show up as an input port, but it's not, it's showing up as an output port instead.  Tracy's code is written correctly, but the underlying system isn't handling it correctly.

Reply:
Oh, my bad - it looks like Tracy fixed the usage based on the next comment below.  Does that make this issue go away as well?  I'm assuming they were the same bug.

Reply:
Yes it's fixed. Thank you, Ben!